Doc/c-api/stable.rst

```diff
@@ -19,7 +19,9 @@ So, code compiled for Python 3.10.0 will work on 3.10.8 and vice versa,
 but will need to be compiled separately for 3.9.x and 3.10.x.
 
 Names prefixed by an underscore, such as ``_Py_InternalState``,
-are private API that can change without notice even in patch releases.
+are private API that can change without notice even in patch releases. ABI
+resulting from it, however, should stay stable across patch versions, unless
+there's critical reason to break compatibility (eg. security issue).
```

Comment:
Tiny note: You could make the sentence start on the next line, to avoid «touching» the existing line 22 and changing the git blame info for it.

This is minor here, but for other PRs in general it’s good to avoid changing (like re-wrapping) lines that don’t have content changes, so that git history stays useful for future investigations.